app/public/js/stream/streamCtrl.js

```diff
@@ -36,4 +47,14 @@ app.controller('StreamCtrl', function ($scope, SCapiService, $rootScope) {
             });
     };
 
+    function markLikedTracks (tracks) {
+        var tracksData = tracks.collection || tracks;
+        for (var i = 0; i < tracksData.length; ++i) {
+            var track = tracksData[i].origin;
+
+            if (track.hasOwnProperty('user_favorite'))
+                track.user_favorite = ($scope.likes.indexOf(track.id) != -1);
```

Comment:
@Pitros the bug here is that you are missing the key `collection` so should be `($scope.likes.collection.indexOf(track.id) != -1)`

if you think both cases are in need with and without the `collection` key you can do a shorthand if.
`$scope.likes.collection ? ($scope.likes.collection.indexOf(track.id) != -1) : ($scope.likes.indexOf(track.id) != -1)`